### ISSUE TITLE
Getting EPIPE error when trying to run buildifier from VSCode

### ISSUE THREAD

**Issue by xykivo** [bazel-contrib/vscode-bazel#214]
Buildifer installed using go, under $GOPATH/bin/buildifier
Configured (in settings) "bazel.buildifierExecutable": "$GOPATH/bin/buildifier"

When trying to have buildifier run on on edit. or on safe, getting warnings that buildifier is not found, and then an EPIPE error.

OS: Ubuntu 20.04
VSCode version: 1.48.2

**Comment by gus** [comment]
It seems like you need use a static, absolute path. No vars allowed. Worked for me anyway.

**Comment by cbandera** [comment]
Correct, the env var that you used is probably not available at the time of interpreting your setting. Note that the vscode extension does not run in your normal shell environment. 
I think the easiest solution to this is in fact to resolve your GOPATH and write the absolute path into the setting
